docs/TRANSPORTATION.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: Transportation
+description: Overview of the Transportation Program
+permalink: /transportation
+---
+
+In order to ensure all Detroit residents have access to the COVID-19 testing site, a transportation service was stood up to pick-up residents at their home, take them through the testing site, drop residents back off at their residence, and then sanitize the vehicle.
+
+How it works:
+
+1. Residents that have met the qualification requirements contact the COVID-19 Test Site call center to request an appointment. If a resident requests transportation during this call, the call center representative connects the resident to the transportation call center for the resident to book a ride directly with transportation vendor.
```

Comment:
the call center representative connects the resident to the transportation call center to book a ride directly with transportation vendor.